roles/core/dns_server/templates/reverse.j2

```diff
@@ -25,7 +25,7 @@ $INCLUDE "{{ dns_server_named_dir }}/{{ item }}.rr.soa"
   {% endif %}
   {% if host_dict['bmc'] is defined %}
     {% set bmc_args = host_dict['bmc'] %}
-    {% if bmc_args.name is defined and bmc_args.name is not none and bmc_args.ip4 is defined and bmc_args.ip4 is not none and bmc_args.network is defined and bmc_args.network is not none and bmc_args.network == outer_item and bmc_args.ip4.split('.')[0:3]|join('.') == item  %}
+    {% if bmc_args.name is defined and bmc_args.name is not none and bmc_args.network is defined and bmc_args.network is not none and bmc_args.network is defined and bmc_args.network.is_in_dns and bmc_args.ip4 is defined and bmc_args.ip4 is not none and bmc_args.ip4.split('.')[0:3]|join('.') == item  %}
```

Comment:
Considering error in Ubuntu global run, I guess here you miss a check on is_in_dns (if defined probably)

Reply:
https://github.com/bluebanquise/bluebanquise/runs/6905696083?check_suite_focus=true

Reply:
Change committed.